src/local.rs

```diff
@@ -278,7 +327,7 @@ async fn prediff(
         app_instance.spec.package.image = package_image.clone();
     }
 
-    write_script(
+    write_apply_script(
```

Comment:
Renamed because `write_script` was really only for the `kubecfg show` (render) + `kubectl apply` steps.

Now we have separate flows dependent on whether an `Apply` or `Cleanup` (deletion) event occurs.